index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
```

Comment:
Please use a semantic header element for your major website title.
To make your code reusable, always use class over id. Kindly change the id attribute to the class attribute.

Reply:
- [ ] Great review @Profsain, however, your comment is a little bit confusing, you could, for instance, rephrase it to a comment like this `Please use a semantic header tag in your title also change your attribute from id to class, this will make your CSS code reusable.`